src/main/resources/liquibase/harness/change/changelogs/postgresql/9.5/datatypes.network.xml

```diff
@@ -5,10 +5,14 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
     <changeSet id="1" author="r2">
-        <createTable tableName="datatypes12_test_table">
-            <column name="serial" type="serial"/>
-            <column name="bigserial" type="bigserial"/>
-            <column name="macaddr8" type="macaddr8" />
+        <createTable tableName="datatypes.network_test_table">
+
+		    <!-- NETWORK TYPES -->
+            <!-- https://www.postgresql.org/docs/12/datatype-net-types.html -->
```

Comment:
plz change version to corresponding (9.5 in this case) or delete this comment

Reply:
+